container/mesos/client.go

```diff
@@ -0,0 +1,158 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
Does this need to be Google?

Reply:
Good question. All the files in this repo have it as "Google", so had it this way. Can check with David on this ?

Reply:
A quick grep on the cadvisor directory shows this to be only "Google"
`grep -r "Copyright" * | grep -v "Google" | grep -v "vendor" | less`

Still waiting on reply from David.

Reply:
I got a reply from David, suggesting that all the file headers should have the copyright associated with "Google"